services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py

```diff
@@ -53,11 +45,12 @@ async def _startup() -> None:
     return _startup
 
 
-def on_app_shutdown(app: FastAPI) -> Callable[[], Awaitable[None]]:
+def on_app_shutdown(
+    app: FastAPI,  # pylint: disable=unused-argument
```

Comment:
This is code-smell (see sonarcloud)

TIP: just ``assert app # nosec `` inside or rename with `_` prefix as `_app` (I prefer the former)

Reply:
thanks!